lang/en/texts/new-nutriscore.html

```diff
@@ -17,8 +17,8 @@ <h2 class="title-2 emphasized-title">
     <strong>Why is Nutri-Score changing?</strong>
 </h2>
 
-            <img width="200px" src="https://world.openfoodfacts.org/images/misc/new-nutriscore/Nutri-Score%20facts%20-%20EN.png" alt="Nutri-Score facts">
-            <img width="200px" src="https://world.openfoodfacts.org/images/misc/new-nutriscore/Why%20a%20new%20algorithm%20-%20EN.png" alt="Why a new algorithm">
+<img width="200px" src="https://world.openfoodfacts.org/images/misc/new-nutriscore/Nutri-Score%20facts%20-%20EN.png" alt="Nutri-Score facts">
```

Comment:
Why 200px? Those images are text heavy, and they are unreadable at that size, both on desktop and mobile.

Reply:
Lack of time for anything better between friday and now